web/testing/index.html

```diff
@@ -19,7 +19,7 @@
     </style>
   </head>
   <body>
-  <h1>KeymanWeb 14 Testing</h1>
+  <h1>KeymanWeb 15 Testing</h1>
```

Comment:
The main complaint I have with this specific change is that we'd have to manually revert this tidbit in the 🍒-pick.

Reply:
If we worry about it ... 😆

Reply:
FWIW, I am going to have to do a careful cherry-pick anyway if we do go ahead, because I don't want to publish the Settings UI changes in 14.0, so I don't mind doing this as well.